src/mpl/configure.ac

```diff
@@ -113,6 +113,11 @@ if test "$enable_g_mem" = "yes" ; then
    AC_DEFINE([USE_MEMORY_TRACING],[1],[Define to enable memory tracing])
 fi
 
+AC_ARG_ENABLE(gpu,
+    AC_HELP_STRING([--enable-gpu], [Build GPU support in MPL (default is no)]),
+    [use_gpu=yes],
+    [use_gpu=no])
+AM_CONDITIONAL([MPL_USE_GPU],[test "x${use_gpu}" = "xyes"])
```

Comment:
Here `AM_CONDITIONAL` will set `MPL_USE_GPU` to be `True` or `False`
I thought we only need this `MPL_USE_GPU` defined, should use AC_DEFINE?

Reply:
This should be removed. I probably missed it in rebasing.